runtime/src/main/cpp/KString.cpp

```diff
@@ -773,7 +773,7 @@ OBJ_GETTER(Kotlin_String_plusImpl, KString thiz, KString other) {
   RETURN_OBJ(result->obj());
 }
 
-OBJ_GETTER(Kotlin_String_toUpperCase, KString thiz) {
+OBJ_GETTER(Kotlin_String_uppercase, KString thiz) {
```

Comment:
It would be a more scoped commit if we don't change cpp symbol names for now, and just implement new functions in terms of the existing ones. It is possible?

Reply:
done